ts/screens/authentication/LandingScreen.tsx

```diff
@@ -32,13 +32,21 @@ type OwnProps = {
 };
 
 type Props = ReduxProps & OwnProps;
+const isNFCavailable = true; // waiting for sdk cie implementation
+const isCIEAvailable = isNFCavailable && isCIEauthenticationEnabled;
 
 const cardProps: ReadonlyArray<ComponentProps<typeof LandingCardComponent>> = [
```

Comment:
If `CIE_AUTHENTICATION_ENABLED = YES`, it is displayed also the following card:

<img width="273" alt="image" src="https://user-images.githubusercontent.com/38431762/67753548-2a377900-fa35-11e9-8082-62360f69752d.png">

Should it be updated when the login by CIE is available? 
cc @matteodesanti @Undermaken

Reply:
@valentinacala @matteodesanti 
If CIE is not available nothing about it should be visible
It could be confusing